src/menu/menu.js

```diff
@@ -28,6 +28,7 @@ export default function Menu(props: StatelessMenuPropsT) {
     focusMenu = () => {},
     unfocusMenu = () => {},
     handleMouseLeave = () => {},
+    handleKeyDown = (event: FocusEvent | MouseEvent | KeyboardEvent) => {},
```

Comment:
Does this need all of the other event types?
```suggestion
    handleKeyDown = (event: KeyboardEvent) => {},
```